src/lib/app/base.ts

```diff
@@ -3,8 +3,7 @@ import { ApiVersion, ChannelClient, IdentityClient } from '@iota/is-client'
 import { derived, get } from 'svelte/store'
 import { logout } from './identity'
 import { showNotification } from './notification'
-import { NotificationType, UserRoles } from './types'
-import type { JwtUser } from './types'
+import { NotificationType, UserRoles, type JwtUser } from './types'
```

Comment:
`type JwtUser` returns an error for me I don't know why not for you? Do you have an idea?

Reply:
```
import { NotificationType, UserRoles } from './types'
import type { JwtUser } from './types'
```
Works for me